### PR TITLE
Touch unet if base model cfg needed for SD pipeline init

### PR DESCRIPTION
**Pull request by monorimet**

### CODE REVIEW

File: apps/stable_diffusion/src/pipelines/pipeline_shark_stable_diffusion_utils.py [hunk 1]
```diff
@@ -91,13 +97,15 @@ def unload_clip(self):
 
     def load_unet(self):
         if self.unet is not None:
+            self.tokenizer = get_tokenizer()
```
Comment (Abhishek-Varma) [inline apps/stable_diffusion/src/pipelines/pipeline_shark_stable_diffusion_utils.py:100]: Not needed.

**Review by powderluv — DISMISSED** [review]

File: apps/stable_diffusion/src/pipelines/pipeline_shark_stable_diffusion_utils.py [hunk 2]
```diff
@@ -91,13 +97,15 @@ def unload_clip(self):
...
             return
 
         if self.import_mlir or self.use_lora:
             self.unet = self.sd_model.unet()
         else:
             try:
                 self.unet = get_unet()
+                self.tokenizer = get_tokenizer()
```
Comment (Abhishek-Varma) [inline apps/stable_diffusion/src/pipelines/pipeline_shark_stable_diffusion_utils.py:108]: Likewise.

**Review by powderluv — COMMENTED** [review]
Will defer to @Abhishek-Varma

**Review by Abhishek-Varma — APPROVED** [review]
LGTM!

**Pull request merged**